README.md

```diff
@@ -0,0 +1,25 @@
+# terra-axon-examples
+
+## Overview
+
+This repository contains example notebooks and documentation for working with [Enterprise Terra](https://verily.com/solutions/terra/).
+
+[Enterprise Terra](https://verily.com/solutions/terra/) is a new platform developed by Verily to connect the multimodal data ecosystem and meet the needs of commercial or industry-led research organizations. Building on Terra’s Academic roots, Enterprise Terra is an integrated platform that provides the secure foundation to discover, govern, and unify multimodal data into a collaborative environment to accelerate discoveries.
```

Comment:
lowercase 'A' in 'Academic'.  'a secure foundation' instead of 'the secure foundation'.  
'in' rather than 'into'